frontend/package.json

```diff
@@ -66,11 +70,23 @@
     "yaml": "^2.1.3"
   },
   "devDependencies": {
+    "@babel/core": "^7.21.0",
+    "@storybook/addon-a11y": "^6.5.16",
```

Comment:
Oooooh, nice! How much is this going to help us?

Reply:
A ton. It essentially runs a11y tests on any "story" we create. So as logn as we contunie to create stories for new (or old) components, this test can be run. 

we can use it in parralell with dev by having the storybook run in watch mode.
we can add it to the storybook testing cli, so both interaction and accessibility tests run
we can have it then integrate with CI